### PR TITLE
Optimized Image relations - Updated HashMap in SpRLSensors

### PR DESCRIPTION
**Pull request by umarmanzoor**
Optimized Image relations - Updated HashMap in SpRLSensors

### CODE REVIEW

**Comment by Rahgooy** [comment]
@kordjamshidi merge?

File: saul-examples/src/main/java/edu/illinois/cs/cogcomp/saulexamples/data/CLEFImageReader.java [hunk 1]
```diff
@@ -251,51 +253,56 @@ private void getSegmentsRelations(String spatial_rels) throws IOException
                                 val = (int) topo[x][y];
                                 if (val == 1)
                                     rel = "adjacent";
-                                else if (val == 2)
-                                    rel = "disjoint";
+//                              ****** Ignoring as not required in mSpRL
+//                                else if (val == 2)
+//                                    rel = "disjoint";
```
Comment (kordjamshidi) [inline saul-examples/src/main/java/edu/illinois/cs/cogcomp/saulexamples/data/CLEFImageReader.java:273]: The whole point of the data model is that we should be able to make such filterings using the data model, not changing the reader.

**Comment by Rahgooy** [comment]
@kordjamshidi merge or close?

**Comment by kordjamshidi** [comment]
I think the problem was with the ontology mapping for this PR

**Pull request closed without merging**